projects/1.19/assets/sweetmagic/sweetmagic/lang/zh_cn.json

```
@@ -0,0 +1,1097 @@
+	"block.sweetmagic.green4panel_glass_pane": "十字镶框玻璃板（绿）",
+	"block.sweetmagic.lightbrown4panel_glass_pane": "十字镶框玻璃板（浅棕）",
+	"block.sweetmagic.brown4panel_glass_pane": "十字镶框玻璃板（棕）",
+	"block.sweetmagic.darkbrown4panel_glass_pane": "十字镶框玻璃板（深棕）",
+	"block.sweetmagic.ami_glass_pane": "嵌丝玻璃板",
+	"block.sweetmagic.gorgeous_glass_pane": "玻璃窗板（黑）",
+	"block.sweetmagic.gorgeous_glass_w_pane": "玻璃窗板（白）",
+	"block.sweetmagic.moden_shelf_i": "现代柜（粉红）",
+	"block.sweetmagic.moden_shelf_l": "现代柜（黄绿）",
+	"block.sweetmagic.moden_shelf_o": "现代柜（橙）",
+	"block.sweetmagic.moden_shelf_p": "现代柜（紫）",
+	"block.sweetmagic.moden_shelf_r": "现代柜（红）",
+	"block.sweetmagic.moden_shelf_s": "现代柜（淡蓝）",
+	"block.sweetmagic.moden_shelf_y": "现代柜（黄）",
+	"block.sweetmagic.woodbox": "板条箱",
+	"block.sweetmagic.treasure_chest": "藏宝箱",
+	"block.sweetmagic.moden_rack": "现代货架（白）",
+	"block.sweetmagic.moden_rack_m": "现代货架（棕）",
+	"block.sweetmagic.moden_rack_t": "现代货架（浅棕）",
+	"block.sweetmagic.moden_rack_b": "现代货架（黑）",
+	"block.sweetmagic.wand_pedastal": "法杖基座",
+	"block.sweetmagic.corkboard": "软木展示框",
+	"block.sweetmagic.wallboard": "壁挂展示框",
+	"block.sweetmagic.wallboard_b": "壁挂展示框（黑）",
+	"block.sweetmagic.couter_table_sink_white_brick_r": "白线砖厨房水槽（红）",
+	"block.sweetmagic.iris_creation": "魔力坩埚",
+	"block.sweetmagic.mill": "磨粉机",
+	"block.sweetmagic.oven": "烤箱",
+	"block.sweetmagic.frypan_r": "平底锅（红）",
+	"block.sweetmagic.pot_w": "锅（白）",
+	"block.sweetmagic.bottle": "发酵瓶",
+	"block.sweetmagic.stove": "灶台",
+	"block.sweetmagic.freezer": "冰箱",
+	"block.sweetmagic.freezer_top": "冰箱",
+	"block.sweetmagic.juice_maker": "饮品机",
+	"block.sweetmagic.magiclight": "魔力光源",
+	"block.sweetmagic.twilightlight": "暮光",
+	"block.sweetmagic.glow_light": "荧光灯",
+	"block.sweetmagic.glow_light_o": "荧光灯（橙）",
+	"block.sweetmagic.gorgeous_lamp": "精致荧光灯",
+	"block.sweetmagic.sweetpotato_plant": "红薯植株",
+	"block.sweetmagic.strawberry_plant": "草莓植株",
+	"block.sweetmagic.onion_plant": "洋葱植株",
+	"block.sweetmagic.olive_plant": "橄榄植株",
+	"block.sweetmagic.soybean_plant": "大豆植株",
+	"block.sweetmagic.rice_plant": "水稻植株",
+	"block.sweetmagic.coffee_plant": "咖啡植株",
+	"block.sweetmagic.corn_plant": "玉米植株",
+	"block.sweetmagic.eggplant_plant": "茄子植株",
+	"block.sweetmagic.tomato_plant": "番茄植株",
+	"block.sweetmagic.lettuce_plant": "生菜植株",
+	"block.sweetmagic.cabbage_plant": "卷心菜植株",
+	"block.sweetmagic.mint_plant": "纸薄荷植株",
+	"block.sweetmagic.cornflower": "魔力矢车菊",
+	"block.sweetmagic.lily_valley": "魔力铃兰",
+	"block.sweetmagic.cosmos": "波斯菊",
+	"block.sweetmagic.blackrose": "黑玫瑰",
+	"block.sweetmagic.white_clover": "白苜蓿",
+	"block.sweetmagic.foxtail_grass": "狐尾草",
+	"block.sweetmagic.snowdrop": "雪莲花",
+	"block.sweetmagic.turkey_balloonflower": "洋桔梗",
+	"block.sweetmagic.iberis_umbellata": "珍珠球",
+	"block.sweetmagic.ultramarine_rose": "青木槿",
+	"block.sweetmagic.solid_star": "满天星",
+	"block.sweetmagic.zinnia": "百日菊",
+	"block.sweetmagic.campanula": "风铃草",
+	"block.sweetmagic.primula_polyansa": "多花报春",
```

```suggestion
	"block.sweetmagic.primula_polyansa": "报春花",
```
可以考虑一下
https://www.plantplus.cn/cn/sp/Primula%20vulgaris